pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepOnMasterTest.java

```diff
@@ -96,18 +93,18 @@ public void after() {
     public void maven_build_on_master_with_specified_maven_installation_succeeds() throws Exception {
         loadMavenJarProjectInGitRepo(this.gitRepoRule);
 
-        //@// @formatter:off
-        String pipelineScript = "node() {\n" +
-            "    git($/" + gitRepoRule.toString() + "/$)\n" +
-            "    withMaven(traceability: true, maven: 'apache-maven-3.6.3') {\n" +
-            "        if (isUnix()) {\n" +
-            "            sh 'mvn package'\n" +
-            "        } else {\n" +
-            "            bat 'mvn package'\n" +
-            "        }\n" +
-            "    }\n" +
-            "}";
-        //@// @formatter:on
+        // @// @formatter:off
```

Comment:
maybe we can fix those ( `// @ // @formatter`  => `// @formatter` )

Reply:
done 0709184c3008d94e041e281df5d8fc38d0038f0e